readme.md

```diff
@@ -38,31 +39,33 @@ from aiodynamo.credentials import Credentials
 from aiodynamo.http.aiohttp import AIOHTTP
 from aiohttp import ClientSession
 
+async def main():
     async with ClientSession() as session:
         client = Client(AIOHTTP(session), Credentials.auto(), "us-east-1")
 ```
 
 ### API use
 
 ```py
-        table = client.table("my-table")
-
-        # Create table if it doesn't exist
-        if not await table.exists():
-            await table.create(
-                Throughput(read=10, write=10),
-                KeySchema(hash_key=KeySpec("key", KeyType.string)),
-            )
-
-        # Create or override an item
-        await table.put_item({"key": "my-item", "value": 1})
-        # Get an item
-        item = await table.get_item({"key": "my-item"})
-        print(item)
-        # Update an item, if it exists.
-        await table.update_item(
-            {"key": "my-item"}, F("value").add(1), condition=F("key").exists()
+async def main():
```

Comment:
`client` below is not defined (it's kind of implicitly supposed to be re-used from the examples above), so could you change this function to take a `client: Client` as argument?

Reply:
Sure, done.